src/games/strategy/triplea/ai/proAI/ProNonCombatMoveAI.java

```diff
@@ -331,6 +331,10 @@ private void findUnitsThatCantMove(final Map<Territory, ProPurchaseTerritory> pu
         }
       }
     }
+
+    // Remove any territories that don't get a unit from result
```

Comment:
This comment requires context to understand. Looking at it, I read it as:
`// Remove from result any territories that don't get a unit from result`
Which seems circular and does not make sense. 

I think better would be a comment about *why* this is being done.